components/GeoTriggering.js

```diff
@@ -1,11 +1,11 @@
 import React, { useState, useEffect } from "react";
 import { Button, Platform, Text, View } from "react-native";
-import { useHistory } from "react-router-native";
+import { useNavigate } from "react-router";
 import BluedotPointSdk from "bluedot-react-native";
 import styles from "../styles";
 
 export default function GeoTriggering() {
-  const history = useHistory();
+  const navigate = useNavigate();
```

Comment:
`useHistory` was removed, use `useNavigate` instead ([react-router-dom v6.0.0](https://reactrouter.com/docs/en/v6/upgrading/v5#use-usenavigate-instead-of-usehistory))